docs/alerts.md

```diff
@@ -150,17 +158,17 @@ Elasticsearch will attempt to relocate shards away from a node for [reaching the
     - To allocate shards to this particular node, free up some spaces:
         - Increase disk space on all nodes.
         - If increasing disk space isn’t possible, then add a new data node to the cluster.
-        - If adding a new data node is problematic, then reduce the number of replicas to 1 if it is higher than 1.
-            - Preferably on older indices.
-            - Get the index settings:
+        - If adding a new data node is problematic, then decrease the total cluster redundancy policy
+            - Check current redundancyPolicy
               ```
-              oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name>/_settings?pretty
+              oc edit Elasticsearch -n openshift-logging
               ```
-            - Identify the number of replicas from the output of the above command.
-            - Lower the number of replicas if possible:
+              Note: If using Cluster Logging Custom Resource then:
               ```
-              oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name>/_settings?pretty -X PUT -d '{"index.number_of_replicas":<number_of_replicas>}'
+              oc edit ClusterLogging instance -n openshift-logging
```

Comment:
nit, but ClusterLogging can be shortened to `cl`
https://github.com/openshift/cluster-logging-operator/blob/master/manifests/5.1/logging.openshift.io_clusterloggings_crd.yaml#L14